libImaging/Resample.c

```diff
@@ -107,7 +107,7 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
     if ( ! kk)
         return 0;
 
-    xbounds = calloc(outSize * 2, sizeof(int));
+    xbounds = malloc(outSize * 2 * sizeof(int));
```

Comment:
malloc: just give me mem. calloc: ensure that if zero-initialized.

In case of `kk` we request `outSize * kmax` elements, but assign only `outSize * xmax` elements, where `xmax` <= `kmax`. And it is better than all other elements (between xmax and kmax) would be zero (because letter it is simpler to iterate over all `kk` elements). We can make it manually after [this loop](https://github.com/python-pillow/Pillow/pull/1933/files#diff-b1e715e96fb950759e657d15a0b3f4f9R130) or we can use calloc.

In the case of `xbounds` we always assign values to all elements. It is not required that memory should be zero-initialized.